branchpro/apps/_dash_app.py

```diff
@@ -53,6 +53,134 @@
 </html>
 """
 
+df = pd.DataFrame({
+    'Time': [1, 3, 4, 5],
+    'Incidence Number': [10, 50, 7, 50],
+    'Imported Cases': [1, None, 1, 1],
+    'R_t': [0.5, 2, 1, 2.5]
+})
+
+inc_modal = [
+    dbc.ModalHeader(html.H6(['Incidence Data'])),
+    dbc.ModalBody(
+        [
+            'The data for the incidences comes in either ',
+            html.Span(
+                '.csv',
+                id='csv_inc',
+                style={
+                    'font-weight':
+                        'bold',
+                    'cursor':
+                        'pointer'}
```

Comment:
Is there any reason to use cursor:pointer in these sections? My impression is that it should be used only for clickable items.